Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -26,16 +26,43 @@ internal class Program
         static void Main(string[] args)
         {
             Console.OutputEncoding = System.Text.Encoding.Unicode;
+            Dictionary<string, int> playerResult = new Dictionary<string, int> { };
```

Comment:
empty dictionary is not needed you can assign variable from startGame method below like: `Dictionary<string, int> playerResult = StartGame();`